po/nb.po

```diff
@@ -31681,9 +30911,8 @@ msgid "Transaction Voided"
 msgstr "Transaksjonen er annullert"
 
```

Comment:
Below are the [Orphaned Records](https://wiki.gnucash.org/wiki/Translation#Orphaned_Records), which you can remove.

Reply:
Always some… ;)